src/main/resources/avro/common.avdl

```diff
@@ -353,6 +353,69 @@ record Sequence {
 }
 
 /**
+An abstraction for referring to a genomic region, in relation to some already 
+known reference. This will require some significant rework as we move to graph
+coordinates.  
+*/
+// TODO: Add support for paths through the graph.
```

Comment:
Maybe this should be done now? This really has to be able to represent any contiguous subgraph.

Otherwise, `GenomicFeature` in `sequenceAnnotations.avdl` can't handle some important cases. For example, a genomic element may start on one `Sequence` and ends on a different one. Or a genomic element may be polymorphic (for example, a gene with a SNP), and both versions of the polymorphism need to be annotated as part of the element.